test/infoblox_test.rb

```diff
@@ -90,4 +91,30 @@ def test_remove_ptr_v6
     @provider.expects(:ib_delete).with(Infoblox::Ptr, :ipv6addr => ip)
     @provider.do_remove(ptr, 'PTR')
   end
+
+  def test_wapi_old
+    fqdn = 'test.example.com'
+    record = Infoblox::Arecord.new name: fqdn
+    record.stubs(:delete).returns(record)
+
+    Infoblox.wapi_version = '2.0'
```

Comment:
This is modification of a global variable in test, either stub it or use `ensure` block to set it back to previous value at the and of the test.